docs/workflows.rst

```diff
@@ -241,20 +241,19 @@ translators to translate into additional languages.
 Source strings reviews
 ----------------------
 
-With :ref:`project-source_review` enabled, the review process can be applied on
-the source strings. Once enabled, users can report issues in the source
-strings.  The actual process depends on whether you use bilingual or
-monolingual formats.
+With :ref:`project-source_review` enabled, the review process can be applied for
+source strings. Once enabled, users can report issues with source strings.
+The actual process depends on whether bilingual or monolingual formats are in use.
 
-For monolingual formats, the source string review behaves similarly as with
-:ref:`reviews` - once issue is reported on the source string, it is marked as
+For monolingual formats, source string review functions similarly to
+:ref:`reviews` — once an issue with a source string is reported, it is marked as
 :guilabel:`Needs editing`.
 
-The bilingual formats do not allow direct editing of the source strings (these
-are typically extracted directly from the source code). In this case
+The bilingual formats do not allow direct editing of source strings (these
+are typically extracted directly from the source code). In this case a
 :guilabel:`Source needs review` label is attached to strings reported by
 translators. You should review such strings and either edit them in the source
-or remove the label.
+code, or remove the label.
```

Comment:
```suggestion
material, or remove the label.
```

Reply:
Totally no need. https://github.com/WeblateOrg/weblate/pull/7505#discussion_r852563528

Reply:
`them in` makes it clear what it is that needs editing.

Reply:
TBH I am not quite sure about what the “source” means here now. It seems ambiguous.

Reply:
It is used in case the source strings are not editable in Weblate, so they need to be edited anywhere it is extracted from. This typically means source code for the applications.